docs/pom.xml

```diff
@@ -32,8 +33,14 @@
             <plugin>
                 <groupId>org.asciidoctor</groupId>
                 <artifactId>asciidoctor-maven-plugin</artifactId>
-                <version>${version.asciidoctor}</version>
+                <version>${asciidoctor-maven-plugin.version}</version>
                 <configuration>
+                    <enableVerbose>true</enableVerbose>  
+                    <logHandler>
+                        <failIf>
+                          <severity>INFO</severity>
```

Comment:
Do we really want to fail where we only have a message at the INFO level? It seems a bit risky?

Reply:
It can sound a bit scary, but the broken cross-reference links are reported at INFO level today by asciidoctor (when enabling verbose) - I think the idea was to experiment with false positives, but I haven't seen any in other projects as well.
On the other hand, broken cross-reference are both:
- easy to introduce (renaming a section or block and forgetting to update the references, possibly in other pages)
- frustrating for users (the ones fixed were on https://quarkus.io/guides/security-guide with links appearing as [xxx])

Though let me know if you prefer I downgrade to WARN, as a first step.

Reply:
Let's do it your way and we will change it if it becomes annoying.